config/routes.rb

```diff
@@ -13,14 +13,23 @@
       end
     end
   end
+
+  resources :save_post do
+    member do
+      patch :save
+      patch :unsave
+    end
+  end
+
+  # get "/saved_posts" => "save_posts#saved_posts"
```

Comment:
remove the commented lines if they are no longer required

Reply:
Removed the commented lines